src/feedback/components/AmountPicker.vue

```diff
@@ -122,9 +95,30 @@ export default {
 </script>
 
 <style scoped lang="stylus">
+$border-radius = 13px
+
+@import '~variables'
+
 .wrapper
   position relative
 
+.content
+  background $grey-2
+  border-radius 13px
```

Comment:
Could use the nice variable defined a few lines above :)

Reply:
ah, I think there were more selectors with this border-radius in our first attempt. Now it's only one appearance. I think we can ditch defined variable in this case :)